src/applications/appeals/995/components/4142/PrivacyActStatementContent.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export const PrivacyActStatementContent = () => (
+  <>
+    <p>
+      <strong>Respondent Burden:</strong> We need this information and your
+      written authorization to obtain your treatment records to help us get the
+      information required to process your claim. Title 38, United States Code,
+      allows us to ask for this information. You can provide this by signing VA
+      Form 21-4142. Federal law permits sources with information about you to
+      release that information if you sign a single authorization to release all
+      your information from all possible sources. We will make copies of it for
+      each source. A few States, and some individual sources of information,
+      require that the authorization specifically name the source that you
+      authorize to release personal information. In those cases, we may ask you
+      to sign one authorization for each source and we may contact you again if
+      we need you to sign more authorizations. We estimate that you will need an
+      average of 10 minutes to review the instructions, find the information and
+      complete this form. VA cannot conduct or sponsor a collection of
+      information unless a valid OMB control number is displayed. Valid OMB
+      control numbers can be located on the{' '}
+      <va-link
```

Comment:
![Screenshot 2025-06-02 at 4 14 54 PM](https://github.com/user-attachments/assets/7d3d8e33-9ddf-479e-bf49-04d9ead291fa)
From the screenshot you posted of the dev build, looks like there might be some strange formatting going on here?

Reply:
Gotcha, thanks for the clarification!